project/BinaryIncompatibilities.scala

```diff
@@ -41,17 +41,7 @@ object BinaryIncompatibilities {
       ProblemFilters.exclude[MissingMethodProblem](
           "scala.scalajs.runtime.RuntimeLong.SIGN_BIT_VALUE"),
       ProblemFilters.exclude[MissingMethodProblem](
-          "scala.scalajs.runtime.RuntimeLong.TWO_PWR_15_DBL"),
-
-      // Synthetic methods inside RuntimeLong, not an issue
-      ProblemFilters.exclude[MissingMethodProblem](
-          "scala.scalajs.runtime.RuntimeLong.scala$scalajs$runtime$RuntimeLong$$isZero"),
```

Comment:
Why are these one not an issue anymore? Is it some fix in `sbt-mima-plugin 0.1.8` or is it something that got fixed along the way through another commit?